### PR TITLE
updating the chiclets to show an icon with ban/add

### PR DESCRIPTION
**Pull request by jenbreese**
Updating the selected facets layout.

### CODE REVIEW

File: app/templates/ui/styles/main.less [hunk 1]
```diff
@@ -12,9 +12,8 @@
 @import "blues-variables.less";
 */
 
-/* MLU design
+/* MLU design */
```
Comment (grtjn) [inline app/templates/ui/styles/main.less]: Looks like you are enabling MLU design by default. Was that intentional?

File: app/templates/ui/styles/main.less [hunk 2]
```diff
@@ -67,6 +66,33 @@ nav.nav {
...
+}
+
+/* negated and added chiclets */
+.chiclet  {
+    position: relative;
+}
+/* set up the added one first */
+.chiclet:before {
+    content: "\f055";
+    font-family: FontAwesome;
```
Comment (grtjn) [inline app/templates/ui/styles/main.less]: This is a less file, right? Can't you pull in fa classes here? `.chiclet { .fa; .fa-whatever; }`?

Comment (jenbreese) [inline app/templates/ui/styles/main.less]: I'm not sure what you mean @grtjn ?

File: app/templates/ui/styles/main.less (hunk 1 above)
Comment (jenbreese) [inline app/templates/ui/styles/main.less]: No that is an accident. I will fix.

File: app/templates/ui/styles/main.less (hunk 2 above)
Comment (grtjn) [inline app/templates/ui/styles/main.less]: I meant mixins I think. See this example: http://stackoverflow.com/a/1065476/918496

**Comment by grtjn** [comment]
Merging for now to at least have the UI changes applied..

**Pull request merged**